### DISCUSSION TITLE
How to create generation_config.json file for finetuned model

### DISCUSSION THREAD

**Discussion by RRThivyan** [facebookresearch/seamless_communication#516] [Q&A]
Hi,

I have finetuned the seamlessm4t_v2_large for indic languages, those not supported for TTS. But only pt file has been generated. How can I get the generation_config.json, config.json etc?

**Comment by deepak-kumar-98** [comment]
@RRThivyan can u pls share, from where you got speaker parallel data for speech to speech?
